misk-web/web/packages/@misk/core/src/ducks/index.ts

```diff
@@ -1,8 +1,28 @@
+import { fromJS, List } from "immutable"
 export * from "./simpleDucksUtilities"
 export * from "./routerDucks"
 export * from "./simpleFormDucks"
 export * from "./simpleNetworkDucks"
 
+// from @misk/common/src/index.ts
```

Comment:
what does this mean?

Reply:
This code used to live in `@misk/common`

Reply:
and it no longer lives there? I think this comment adds more confusion than clarity.